src/shared/lib/resources/affiliation.ts

```diff
@@ -26,8 +26,15 @@ export interface Affiliation {
 // Used when returning a list of the current user's affiliations
 export interface AffiliationSlim {
   id: Id;
+  membershipType: MembershipType;
   organization: Pick<Organization, 'id' | 'legalName'>;
+}
+
+export interface AffiliationMember {
+  id: Id;
   membershipType: MembershipType;
+  membershipStatus: MembershipStatus;
+  user: Pick<User, 'id' | 'name' | 'avatarImageFile' | 'capabilities'>;
 }
```

Comment:
I think you can use `AffiliationMember` as the new `AffiliationSlim` and avoid needing two separate types for the read many action in the affiliation resource. Unless I'm missing something?

Reply:
I did it this way because `AffiliationSlim` includes the org, so `rawAffiliationToAffiliationSlim` uses `readOneOrganization`, which in turn invokes `calculateSWUQualifiedStatus` causing a loop.

`AffiliationSlim` is used for getting a list of orgs a user belongs to so needs to include org.
`AffiliationMember` is used for getting a list of users belonging to an org so does not need to include org.

Reply:
Got it — thanks for the explanation